test/libsolidity/semanticsTests/EndToEnd/return_string.sol

```diff
@@ -0,0 +1,20 @@
+contract Main {
+    string public s;
+    function set(string _s) external returns (bool) {
+        s = _s;
+        return true;
```

Comment:
This now returns a bool, since "no return data" will be considered the same as "revert" - so far I haven't found a reliable way to distinguish between the two using the cpp-ethereum JSON RPC interface.